packages/cli/src/commands/new/python.ts

```
@@ -0,0 +1,156 @@
+            );
+          }
+
+          return true;
+        })
+        .check(({ name }) => {
+          if (fs.existsSync(path.join(process.cwd(), name))) {
+            throw new Error(`"${name}" already exists!`);
+          }
+
+          if (!/^(?:@[a-z\d\-*~][a-z\d\-*._~]*\/)?[a-z\d\-~][a-z\d\-._~]*$/.test(name)) {
+            throw new Error(`"${name}" is not a valid package name`);
+          }
+
+          return true;
+        });
+    },
+    handler: async ({ name, template, atk, start, clientId, clientSecret }) => {
+      const projectDir = path.join(process.cwd(), name);
+      const builder = Project.builder()
+        .withPath(projectDir)
+        .withName(name)
+        .withLanguage('python')
+        .addTemplate(template);
+
+      if (atk) {
+        builder.addAgentsToolkit(atk);
+      }
+
+      if (clientId) {
+        builder.addEnv('CLIENT_ID', clientId);
+      }
+
+      if (clientSecret) {
+        builder.addEnv('CLIENT_SECRET', clientSecret);
+      }
+
+      if (process.env.OPENAI_API_KEY) {
+        builder.addEnv('OPENAI_API_KEY', process.env.OPENAI_API_KEY);
+      }
+
+      if (process.env.AZURE_OPENAI_API_KEY) {
+        builder.addEnv('AZURE_OPENAI_API_KEY', process.env.AZURE_OPENAI_API_KEY);
+      }
+
+      if (process.env.AZURE_OPENAI_ENDPOINT) {
+        builder.addEnv('AZURE_OPENAI_ENDPOINT', process.env.AZURE_OPENAI_ENDPOINT);
+      }
+
+      const project = builder.build();
+      await project.up();
+      console.log(
+        new String()
+          .bold(new String().green(`✅ App "${name}" created successfully at ${projectDir}`))
+          .toString()
+      );
+
+      if (start) {
+        console.log(`cd ${name} && poetry install && poetry run python src/main.py`);
+        cp.spawnSync(`cd ${name} && poetry install && poetry run python src/main.py`, {
+          stdio: 'inherit',
+          shell: true,
+        });
```

Hmm. So this kind of forces/assumes "poetry" being present.

We need to:
1. Endorse some tool (poetry? or uv?).
2. We _check_ if that tool exists. If not, then we should throw an error saying we can't start.
3. We also need to activate the virtual environment.